Looms.PoS/Looms.PoS.Application/Models/Requests/CreateDiscountRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Looms.PoS.Domain.Enums;
+
+namespace Looms.PoS.Application.Models.Requests;
+
+public record CreateDiscountRequest
+{
+    public string? Name { get; init; } = string.Empty;
+    public DiscountType DiscountType { get; init; }
```

Comment:
Change enum types to string and validate in validator

Reply:
changed to string for both request and response

Reply:
response should be the correct type but make sure it outputs the string value and not a number

Reply:
changed back to enums in response